docs/configuration.rst

```diff
@@ -1663,6 +1663,26 @@ Example:
    The created ``needs.json`` file gets stored in the ``outdir`` of the current builder.
    So if ``html`` is used as builder, the final location is e.g. ``_build/html/needs.json``.
 
+.. _needs_build_needumls:
+
+needs_build_needumls
+~~~~~~~~~~~~~~~~~~~~
+
+Creates a folder during other builds, like ``html``.
+
+This option like :ref:`needs_build_json` allows to have one single Sphinx-Build for two output formats. 
+
+All the files that stores the generated plantuml code from :ref:`needuml` will be saved in this created folder
+in the ``outdir`` of the current builder, e.g. ``_build/html/my_needumls/`` 
```

Comment:
If you use "my_needumls" here, you should show the example first, as it tells the user that "my_needumls" is not a hard configured name but can be configured by the user.